src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -77,10 +64,15 @@ def transform_gene_info(
     )
 
     druggability = nest_fields(
-        df=druggability,
+        df=(
```

Comment:
We are keeping the current data structure because that is the data structure that the downstream application expects. This allows us to avoid a refactor that we don't have time to take on.